src/services/player/player.ts

```diff
@@ -0,0 +1,16 @@
+import TrackPlayer, { Event } from 'react-native-track-player'
+
+module.exports = async function () {
```

Comment:
```suggestion
export async function () {
```

Reply:
It doesn't work that way. Give an error message.